src/Native/Runtime/thread.h

```diff
@@ -135,7 +135,7 @@ class Thread : private ThreadBuffer
     void ClearState(ThreadStateFlags flags);
     bool IsStateSet(ThreadStateFlags flags);
 
-    static UInt32_BOOL HijackCallback(HANDLE hThread, PAL_LIMITED_CONTEXT* pThreadContext, void* pCallbackContext);
+    static UInt32 HijackCallback(HANDLE hThread, PAL_LIMITED_CONTEXT* pThreadContext, void* pCallbackContext);
```

Comment:
Why have you changed the return type? The function returns TRUE / FALSE, so the return type was correct.

Reply:
This was to match typedef in the RedHawk.h header, where we have UInt32: https://github.com/dotnet/corert/pull/4637#discussion_r142313371.

Reply:
Ah, I see. Ok, then I would rather fix it the other way - change the return type in the header to UInt32_BOOL. @jkotas since you have suggested the other way, are you ok with that?

Reply:
Yes, fixing this other way is fine too.

Reply:
Thanks for the confirmation, I have fixed up all the feedback in 89a1373.